clients/rancher/client.go

```diff
@@ -86,6 +90,25 @@ func NewClient(bearerToken string, session *session.Session) (*Client, error) {
 
 	c.Steve.Ops.Session = session
 
+	if rancherConfig.CLIVersion != "" {
+		projectConfig := &management.Project{
+			ClusterID: "local",
+			Name:      namegen.AppendRandomString("project"),
+		}
+
+		testProject, err := c.Management.Project.Create(projectConfig)
+		if err != nil {
+			return nil, err
+		}
+
+		c.CLI, err = ranchercli.NewClient(session, rancherConfig.CLIVersion, bearerToken, rancherConfig.Host, testProject.ID)
+		if err != nil {
+			return nil, err
+		}
+
+		c.CLI.Session = session
```

Comment:
If you pass the session you don't need to do this Markus

Reply:
Heard, I was following the examples above, but that's a good point. I'll remove it

Reply:
Resolved.